helm/cosmo/charts/studio/templates/config-map.yaml

```diff
@@ -13,3 +13,4 @@ data:
   studioWebUrl: "{{ .Values.global.studio.webUrl }}"
   slackClientId: "{{ .Values.configuration.slackClientId }}"
   customHeadScripts: "{{ .Values.configuration.customHeadScripts }}"
+  stripePublishableKey: "{{ .Values.configuration.stripePublishableKey }}"
```

Comment:
Is this a secret? If yes, please put it in the secret map.

Reply:
No it's public